src/style.css

```diff
@@ -0,0 +1,52 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars .stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
```

Comment:
add default background-position, background-repeat and background-image (not active star) to each star (.stars .stars__star) and then just rewrite background-image with active star for each modifier

Reply:
@aliceblunt8 Done